factory/consensus/consensusComponents.go

```diff
@@ -270,7 +270,16 @@ func (ccf *consensusComponentsFactory) Create() (*consensusComponents, error) {
 		return nil, err
 	}
 
+	var log logger.Logger
+	if ccf.flagsConfig.WithInstanceLogID {
```

Comment:
this code is duplicated in many places.. maybe extract it as a method in a common place, with 3 params, the boolean from config, the base log string and the pub key?